packages/esify/README.md

```diff
@@ -15,3 +15,37 @@ From the root of the Shopify directory, run this script with a single, relative
 ```sh
 esify app/assets/javascripts/admin/lib/*.coffee
 ```
+
+You can provide custom options to `esify` by adding an `esify.config.js` file to the directory from which you are running the `esify` command. An example configuration is shown below:
+
+```js
+// your-project/esify.config.js
+var path = require('path');
+
+module.exports = {
+  // The global namespaces used in your current JavaScript code.
+  appGlobalIdentifiers: ['Shopify'],
+
+  // The root folder for your JavaScripts
+  javaScriptSourceLocation: path.join(__dirname, 'app/assets/javascripts'),
+
+  // The output style for your code
+  printOptions: {
```

Comment:
It'd be useful to link to https://github.com/benjamn/recast/blob/master/lib/options.js here.